src/sentry/static/sentry/app/views/settings/organizationSecurityAndPrivacy/organizationSecurityAndPrivacyContent.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import {RouteComponentProps} from 'react-router/lib/Router';
+import PropTypes from 'prop-types';
+
+import {t} from 'app/locale';
+import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
+import JsonForm from 'app/views/settings/components/forms/jsonForm';
+import Form from 'app/views/settings/components/forms/form';
+import AsyncView from 'app/views/asyncView';
+import {Organization} from 'app/types';
+import {addErrorMessage} from 'app/actionCreators/indicator';
+import {updateOrganization} from 'app/actionCreators/organizations';
+import organizationSecurityAndPrivacy from 'app/data/forms/organizationSecurityAndPrivacy';
+
+export type OrganizationSecurityAndPrivacyContentProps = {
+  organization: Organization;
+  params: {
+    orgId: string;
+    projectId: string;
+  };
+} & RouteComponentProps<{}, {}>;
+
+class OrganizationSecurityAndPrivacyContent extends AsyncView<
+  OrganizationSecurityAndPrivacyContentProps
+> {
+  static contextTypes = {
```

Comment:
`router` doesn't seem to be used?

Reply:
yes but if I don't pass it here, tests won't pass... there is some issue with propTypes

Reply:
@billyvg  ![image](https://user-images.githubusercontent.com/29228205/76068674-dcf28780-5f91-11ea-978a-76eeed8bd53f.png)

Reply:
funny thing....I cannot remove the organization as well...otherwise I break unit test

just tested it now